pipeline/data/download-mono.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+"""
+Downloads a monolingual dataset, shuffles it, and truncates it to a maximum amount of sentences.
+
+Kinds:
+   taskcluster/kinds/dataset/kind.yml
+
+Example usage:
+
+    pipeline/data/download-mono.py                  \\
+        --dataset news-crawl_news.2021              \\
+        --language en                               \\
+        --max_sentences 100000000                   \\
+        --artifacts $TASK_WORKDIR/artifacts
+
+Artifacts:
+
+    artifacts
+    └── news.2021.en.zst
+"""
+
+import argparse
+import gzip
+import io
+import os
+import re
+from random import Random
+from typing import Iterator, Optional
+
+import requests
+import zstandard
+from google.cloud.storage.fileio import BlobReader
+
+from pipeline.utils import Dataset, attempt_mocked_request
```

Comment:
The code in the `pipeline` directory should not refer to the `utils` directory. `utils` is reserved for local helper cli tools. If we ever want to introduce code that's used by both pipeline and some other functionality, we can create a new top level directory like `common`. I can imagine logging and some other shared things live there.